hack/patch_node.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+PROXY_PORT=9999
+
+function finish {
+  rc=$?
+  if jobs 1 | grep -q Running; then
+    echo
+    echo "Killing proxy (pid=$proxy_pid)..."
+    kill "$proxy_pid"
+  fi
+  exit "$rc"
+}
+
+# creates/removes prow/sriov resources for the given $node
+function patch_node {
+  local node=$1
+  local capacity=$2
```

Comment:
The docs explain about allocate one or two jobs, here I see that the capacity can vary. 
Maybe we should mention in the doc that it can be more than 2, Or limit here the capacity to 2.

Reply:
Please see https://github.com/kubevirt/project-infra/pull/784#discussion_r568719214 (partially related).
the capacity here means only how many sriov/prow resources there will be,
the README just ask you that if you run the CNI twice, you need to request 2 resources out of the allocatable ones.
the capacity here is the configuration you apply to the node, based on how many PFs it physically have, and should not be limited.

Reply:
If it is not limited to 2,  maybe you can generalize the explanation in the doc?

Reply:
The document doesn't explain the script, it explains only how to allocate more than one PF.

The README is part of this PR, just because the prow/sriov is introduced in this PR.
There is no other scenarios that we will need beside using one of 2 PFs, and even if there will be, one can derive them from the examples there.

I will add the the script itself some guidelines about the script usage itself if needed.

Reply:
OK. Thanks.